libkbfs/keybase_service_base.go

```diff
@@ -377,7 +384,10 @@ func (k *KeybaseServiceBase) PaperKeyCached(ctx context.Context,
 		}
 		// Ignore any errors for now, we don't want to block this
 		// notification and it's not worth spawning a goroutine for.
-		k.config.MDServer().CheckForRekeys(context.Background())
+		mdServer := k.config.MDServer()
+		if mdServer != nil {
```

Comment:
Perhaps retry here as well (maybe in a new goroutine), since otherwise it could delay rekeys?